.travis.yml

```diff
@@ -9,54 +9,38 @@ dist: trusty
 
 env:
   global:
-    - EMULATOR_API=24
+    - EMULATOR_API=16
```

Comment:
How come we are dropping the API back so much?

Reply:
The tracker supports SDK 14 at the minimum and emulator startup time of 16 is lower than 24 in Travis. It reduced build time from ~30 min to ~22-23 min. Do you think having a recent version will bring an advantage?

Reply:
So API version 16 is used only by 1.2% of devices according to this page:

> https://developer.android.com/about/dashboards

I would argue that using a later version is more useful in that you are testing on more relevant releases.

Reply:
I see your point, I'll revert it 👍